opentimelineio/core/composition.py

```diff
@@ -67,10 +67,14 @@ def __init__(
         )
         collections.MutableSequence.__init__(self)
 
+        # used to optimize membership checks on children.
```

Comment:
```suggestion
        # Because we know that all children are unique, we store a set
        # of all the children as well to speed up __contain__ checks.
```